examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "compliant-eligibility",
+   "metadata": {},
+   "source": [
+    "# Complex data type detection using the Data Labeler component of the Data Profiler"
```

Comment:
`Sensitive data` detection using the Data Labeler component of the Data Profiler

`Entity` detection using the Data Labeler component of the Data Profiler

I think `complex data type` is not clear to most people, we can be more specific and pull people in. IMO I'd go with "Sensitive data"

Reply:
changed to sensitive data